recipes/nrtk-explorer/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "nrtk-explorer" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nrtk_explorer-{{ version }}.tar.gz
+  sha256: 05a0c81476df66979b4a6fc23814a6b4e8589573115b910aabf667b8fac092ac
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<39]
+  skip: true  # [osx and py<310]
+
+requirements:
+  host:
+    - hatchling
+    - nodejs
+    - pip
+    - python
+  run:
+    - accelerate
+    - nrtk ==0.4.1
+    - numpy
+    - pillow
+    - python
+    - pytorch
+    - scikit-image
+    - scikit-learn ==1.4.1.post1
+    - scipy >=1.8.1,<1.9  # [py<311]
+    - scipy >=1.9  # [py>=311]
```

Comment:
Change to just
```
scipy >=1.8.1
```
without any preprocessor selectors

Reply:
The problem with this is that while it "builds" it will not pass the tests "import nrtk-explorer" or run since the pybsm pyproject has the following dependency: `scipy = [
    {version = ">=1.8.1,<1.9", python = ">=3.8,<3.11"},
    {version = ">=1.9", python = "^3.11"}
]` scipy >=1.8.1 install something over 1.9

Reply:
Does `nrtk-explorer` not support python 3.12 currently?

Reply:
That is not a problem here, Conda will make packages for each supported python release 3.9/3.10/3.11/3.12. We hit these issues when testing with 3.9

Reply:
Yes it does support 3.12

Reply:
Also note that we do not depend on scipy directly, the reason of adding this is to be a workaround until we fix the pybsm package.